src/app/services/copiedForGitHubCommenting.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { map, flatMap, catchError } from 'rxjs/operators';
+import { throwError, of } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class WorldChampionsService {
+
+  initialYear = 1950;
```

Comment:
Id add a type: `initialYear: number`

Reply:
~~And probably declare it with `let` or `const`  too depending on if you expect to change its value or not.~~ Edit: class properties dont get `var`s, `let`s, `const`s. Duh. (This is why I like code reviews, it forces me to think about the code a little differently than when I am writing it with an IDE)

Reply:
Every comment, take with a grain of salt. Im rusty with this stuff. Just commenting on what comes to mind. :)